flixel/addons/editors/pex/FlxPexParser.hx

```diff
@@ -54,7 +55,7 @@ class FlxPexParser
 		var angle = minMax("angle", config);
 		
 		var startSize = minMax("startParticleSize", config);
-		var finishSize = minMax("finishParticleSize", "FinishParticleSizeVariance", config);
+		var finishSize = minMax("finishParticleSize", "finishParticleSizeVariance", config);
```

Comment:
Looks like you forgot to add the link.. is it this tool? https://71squared.com/particledesigner